problems/[230710] 35. Search Insert Position.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * @param {number[]} nums
+ * @param {number} target
+ * @return {number}
+ */
+const searchInsert = function (nums, target) {
+  let left = 0;
+  let right = nums.length - 1;
+
+  while (left < right) {
+    const centerIdx = Math.floor((right + left) / 2);
+
+    if (nums[centerIdx] < target) {
+      left = centerIdx + 1;
+    } else {
+      right = centerIdx;
+    }
+  }
+
+  if (nums[right] === target) {
+    return right;
+  }
+
+  return nums[right] < target ? right + 1 : right;
```

Comment:
```suggestion
return left;
```

while 문에서 3가지 경우의 수를 다뤄준다면,
target이 없을 때  while 문을 벗어나게 됩니다.
그럼 마지막 left의 값이 결국 target의 자리가 된다고 생각했습니다